Makefile

```diff
@@ -21,8 +24,8 @@ format-style:
 	@# `make` needs `$$` to output `$`. Ref: http://stackoverflow.com/questions/2382764.
 	autopep8 `git ls-files | grep "\.py$$"`
 
-test: clean check-syntax-errors check-style
-	nosetests
+test: clean check-syntax-errors format-style check-style
```

Comment:
⚠️ No, by adding `format-style` here, we are actually not flaking the code anymore in the CI! ⚠️ 

Now [tests pass](https://circleci.com/gh/openfisca/openfisca-core/2745) with [bad style](https://github.com/openfisca/openfisca-core/compare/mess-up-flake) because autopep8 fixes the style before flake8, but the style fix are not commited!

Reply:
Oops ! Thanks for pointing that out @fpagnoux